aiida/manage/configuration/config.py

```diff
@@ -434,7 +434,7 @@ def get_options(self, scope=None) -> Dict[str, Tuple[Option, str, Any]]:
         output = {}
```

Comment:
an explanation of what the `scope` is would be useful (optional)

Reply:
done